src/System.Collections/tests/Generic/Stack/Stack.Generic.Tests.cs

```diff
@@ -53,12 +53,10 @@ protected override IEnumerable<T> GenericIEnumerableFactory(int count)
         protected override bool Enumerator_Current_UndefinedOperation_Throws { get { return true; } }
 
         #endregion
-
-        #region Constructor
-
-        #endregion
-
-        #region Constructor_IEnumerable
+        
+        // Constructor (TODO)
```

Comment:
I simply left a TODO since the `#region Constructor` was empty; I can probably add a few basic tests for the no-args constructor.